docs/etcd-tls-assets.md

```diff
@@ -772,3 +772,52 @@ Eventually, when `cluster-etcd-operator` is installed, running, and
 reporting the `EtcdRunningInCluster = True` condition. This is usually
 when the etcd cluster has scaled to 4, and it is safe to remove the
 bootstrap member.
+
+# The Big CA Cert Rotation
+
+With 4.16 we want to take an initial stab at the rotation of the signer certificates.
+
+The whole process can be split into a two-step migration:
+1. Operator must manage all etcd certificates [CEO#1177](https://github.com/openshift/cluster-etcd-operator/pull/1177) 
+   1. Operator should create new signers, in addition to the current ones in openshift-config. Both bundled and distributed.  
+      This allows us to stop the "death clock" on any existing signer expiration and allowing manual rotation immediately.
+   2. Having the rotation code already in place also enables easier offline rotation [ETCD-510](https://issues.redhat.com/browse/ETCD-510)
+2. Operator must orchestrate an automated signer rotation process
+   1. A new signer needs to be generated, distributed via bundle and static pod rollout
+   2. All certificates must be re-generated using the new signer
+   3. All consumers of client certs (Prometheus, CEO, KAS) need to be notified or must react correctly to a rotated certificate secret
+
+With step (1), we want to move the source-of-truth for etcd certificates into the `openshift-etcd` namespace. This should 
+help in future backup and restore stories, allowing us to more easily collect all certificates required for etcd and guard 
+them better by RBAC.
+
+The above cert summary tables become as follows:
+
+## etcd CA summary
+
+"Old" etcd CAs and their CA bundles remain stored in the `openshift-config` namespace. 
+The new CAs and their bundles are moved to `openshift-etcd`, bundles always contain the old CAs from `openshift-config`. 
+
+| CA (secret)                                      | CA bundle (configmap)                   |
+|--------------------------------------------------|-----------------------------------------|
+| openshift-config/etcd-signer (deprecated)        | openshift-config/etcd-ca-bundle         |
+|                                                  | openshift-etcd/etcd-ca-bundle           |
+| openshift-etcd/etcd-signer (NEW)                 | openshift-etcd/etcd-ca-bundle           |
+| openshift-config/etcd-metric-signer (deprecated) | openshift-config/etcd-metric-serving-ca |
+|                                                  | openshift-etcd/etcd-metrics-ca-bundle   |
+| openshift-etcd/etcd-metric-signer (NEW)          | openshift-etcd/etcd-metrics-ca-bundle   |
+
+The new bundles are then distributed using the `ResourceSyncController` to all consumers, e.g. `openshift-etcd-operator`.
```

Comment:
I'll update the tables accordingly :) I tried to append as much as possible towards the end, because we still need to have some docs for previous versions of OCP.